src/channel.cc

```diff
@@ -13,7 +13,8 @@ ncclResult_t initChannel(struct ncclComm* comm, int channelId) {
   if (channel->id != -1) return ncclSuccess;
 
   int nRanks = comm->nRanks;
-  int nPeers = nRanks + 1 /* Collnet */ + comm->localRanks /* NVLS */;
+  int nvlsRanks = comm->MNNVL ? comm->clique.size : comm->localRanks;
+  int nPeers = nRanks + 1 /* Collnet */ + nvlsRanks /* NVLS */;
```

Comment:
Changes surrounding MNNVL appear in multiple places. Does this imply changes we must make beyond this merge? @wenkaidu @akolliasAMD

Reply:
We can ignore any changes related to MNNVL as it is not supported on AMD GPUs. So just make no change or minimal changes to make further merge easier.